rusk-wallet/src/bin/command.rs

```diff
@@ -545,11 +536,8 @@ impl Command {
             } => {
                 let gas = Gas::new(gas_limit).with_price(gas_price);
 
-                let address = match address {
-                    Some(addr) => wallet.claim(addr)?,
-                    None => wallet.default_address(),
-                };
-                let addr_idx = wallet.find_index(&address)?;
+                let addr = address.unwrap_or(wallet.default_address());
```

Comment:
```suggestion
                let address = address.unwrap_or(wallet.default_address());
```

Shading the original address variable is less error prone imo